airbyte-integrations/bases/base-normalization/normalization/transform_catalog/destination_name_transformer.py

```diff
@@ -231,6 +231,7 @@ def transform_standard_naming(input_name: str) -> str:
 
 def transform_json_naming(input_name: str) -> str:
     result = sub(r"['\"`]", "_", input_name)
+    result = sub(r"\n", "_", result)
```

Comment:
This function reproduces behavior that certain destination connectors do on special property names following the code here: https://github.com/airbytehq/airbyte/blob/85381bde3c0909c1e092ebd82409605431980f74/airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination/StandardNameTransformer.java#L45

So adding this line here is not enough...
and I am not sure about the core issue that is being solved here and why this would be the proper solution?

Do we really expect column names with `\n` characters btw?

Reply:
> Do we really expect column names with \n characters? 

This change may "fix" the SQL syntax error, but is the data properly parsed from the json blob or is it producing empty NULL values for that column? The answer probably boils down to if `\n` is an important character in the column name or not

So we'd need more context to dig deeper:
- example of record data in the raw table 
- catalog.json file to see how the `\n` is appearing there in the property name